src/id/dos33.a

```diff
@@ -62,11 +62,34 @@ IDDOS33
          !byte $6C,$3E,$00
          !byte $EE,$FE,$08
          !byte $EE,$FE,$08
+         !byte WILDCARD,WILDCARD,WILDCARD

```

Comment:
instead of 9 wildcards, merge the "$A6 $2B" into the checks below (1 for DOS 3.3, 1 for Sierra variant)